app/views/users/show.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Show users pages</h1>
\ No newline at end of file
+<h1>User ID: <%= @user.id %> </h1>
+<h1>Name: <%= @user.name %> </h1>
+<img src=<%= @user.photo%> />
+<h3>Number of posts:<%= @user.posts_counter%> </h3>
+<hr/>
+<h4>Bio</h4>
+<p><%= @user.bio %> </p>
+<hr/>
+<% @user.recent_posts.each_with_index do |post, index| %>
+    <a href='/users/<%= @user.id %>/posts'> <h3>Post # <%= index + 1 %></h3></a>
+    <h4><%= post.text %></h4>
+    <h5>Comments:<%= post.comments_counter %></h5>
+    <h5>Likes:<%= post.likes_counter %></h5>
+    <hr/>
+<% end %> 
+<a href='/users/'><button>Go back</button></a>
```

Comment:
- [ ] Same here, Please edit all elements placement to match the given design. 

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/198071695-76bb3452-905f-4d58-b3f1-f47844f22bd4.png)  | ![image](https://user-images.githubusercontent.com/68796232/198071740-41fba5d6-4dd2-4eef-81d8-c682ad2fbaca.png) |